projects/angular/src/layout/tabs/tab-link.directive.ts

```diff
@@ -40,7 +40,7 @@ export class ClrTabLink {
 
   constructor(
     public ifActiveService: IfActiveService,
-    @Inject(IF_ACTIVE_ID) private id: number,
+    @Inject(IF_ACTIVE_ID) private _id: number,
```

Comment:
```suggestion
    @Inject(IF_ACTIVE_ID) public readonly id: number,
```

We can just use `readonly` instead of the get accessor wrapper.

Reply:
@kevinbuhmann , makes more sense, thanks, updated